ai-service/src/ai_service/main.py

```
@@ -1,53 +1,103 @@
+
+        print("Embedding content for each file ...")
+        for file_path in code_files:
+            try:
+                with open(file_path, encoding="utf-8") as f:
+                    code = f.read().strip()
+                    if not code:
+                        print(f"Skipping empty file: {file_path}")
+                        continue
+                    code_snippets.append(code)
+                    embeddings.append(embedder.embed_text(code))
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)